src/main/java/com/javapractices/app/Account.java

```diff
@@ -0,0 +1,22 @@
+package com.javapractices.app;
+
+public class Account {
+	double principal, rate;
+	int daysActive, accountType;
+	public static final int STANDARD = 0, BUDGET = 1, PREMIUM = 2,
```

Comment:
Wouldn't it be more practical to create an enum to store these values?

`public enum AccountTypes {
    STANDARD, BUDGET, PREMIUM, PREMIUM_PLUS
}`

Also it would be nice to modularize this structure in a resource file like Constants.java. Because it can be useful to know these values ​​when implementing the logic of making certain features available for certain types of accounts.